src/instruments/src/EFB/Dashboard/Widgets/WeatherWidget.tsx

```diff
@@ -72,6 +72,9 @@ type WeatherWidgetProps = { name: string, editIcao: string, icao: string};
 const WeatherWidget = (props: WeatherWidgetProps) => {
     const [metar, setMetar] = useState<MetarParserType>(MetarParserTypeProp);
 
+    let getBaroTypeForAirport: string = icao: string => ["K", "C", "M", "P", "RJ", "RO", "TI", "TJ"].some(r => icao.startsWith(r)) ? "IN HG" : "HPA";
```

Comment:
```suggestion
    const getBaroTypeForAirport = (icao: string) => {
        return ["K", "C", "M", "P", "RJ", "RO", "TI", "TJ"].some(r => icao.startsWith(r)) ? "IN HG" : "HPA"
    };
```

Reply:
This is a bit cleaner

Reply:
I see. Still learning JavaScript / TypeScript.